server/notification-providers/yzj.js

```diff
@@ -0,0 +1,76 @@
+const NotificationProvider = require("./notification-provider");
+const { DOWN, UP } = require("../../src/util");
+const { default: axios } = require("axios");
+
+class YZJ extends NotificationProvider {
+    name = "YZJ";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            if (heartbeatJSON != null) {
+                let params = {
+                    content: `${this.statusToString(heartbeatJSON["status"])} ${monitorJSON["name"]} \n> ${heartbeatJSON["msg"]}\n> Time (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`
+                };
+                if (await this.sendToYZJ(notification, params)) {
+                    return okMsg;
+                }
+            } else {
+                let params = {
+                    content: msg
+                };
+                if (await this.sendToYZJ(notification, params)) {
+                    return okMsg;
+                }
+            }
```

Comment:
you are overcomplicating this a bit.

```suggestion
            let msg = msg;
            if (heartbeatJSON != null) {
                msg = `${this.statusToString(heartbeatJSON["status"])} ${monitorJSON["name"]} \n> ${heartbeatJSON["msg"]}\n> Time (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`
            }
            
            // inline sendToYZJ here.. I think this is simpler to read than currently
```